### PR TITLE
[RSS-73] Style Desktop Login Page

### PR DESCRIPTION
**Pull request by buffersquid**
I think everything looks like the mockup. Image provided. Let me know if I should fix anything.
[
Screen Shot 2021-04-18 at 8 58 52 PM
](url)

### CODE REVIEW

**Comment by buffersquid** [comment]
Also, Jack can attest to the npm package issues I was having, and I'm not sure if the fix we had to use will mess up anything in master. So let me know if I should do anything about that.

File: src/components/LoginPage/LoginPage.jsx [hunk 1]
```diff
@@ -14,26 +14,52 @@ const StyledContainer = styled.div`
...
+      padding: 40px;
+   }
 `;
 const StyledImage = styled.img`
    display: flex;
    justify-content: center;
+   @media only screen and (min-width: 768px) {
+      max-width: 200%;
+      height: auto;
+   }
```
Comment (jpoist97) [inline src/components/LoginPage/LoginPage.jsx]: Can we make the max-width of this a specific number? It looks a bit off when the screen gets wider around like 700px. I think whatever the size is for larger screens should do.


Screen Shot 2021-04-18 at 10 18 37 PM

File: src/components/LoginPage/LoginPage.jsx [hunk 2]
```diff
@@ -14,26 +14,52 @@ const StyledContainer = styled.div`
...
 `;
+/*
+@media only screen and (min-width: 768px) {
+      display: block;
+      margin-left: auto;
+      margin-right: auto;
+      width: 347px;
+      height: 220px;
+   }
+*/
```
Comment (jpoist97) [inline src/components/LoginPage/LoginPage.jsx]: I think you can get rid of this

**Review by jpoist97 — COMMENTED** [review]
Looks good, couple of small comments. What kind of package errors were you getting? Also what version of npm do you have installed?

**Comment by buffersquid** [comment]
@jpoist97 We had to add a section in .eslint that made the path case sensitive. But since I was the only one having this strange error, I want to make sure that when my branch gets merged it doesn't, mess up the repo or anything like that.

**Review by jackfales — COMMENTED** [review]
Looks good for the most part. The "Forgot Password" link and Show/Hide Password button look a bit weird on wide screens. I think this has to do with how you're using flexbox for these components.
Screen Shot 2021-04-19 at 6 11 03 PM

**Comment by jpoist97** [comment]
> @jpoist97 We had to add a section in .eslint that made the path case sensitive. But since I was the only one having this strange error, I want to make sure that when my branch gets merged it doesn't, mess up the repo or anything like that.

Hm, I've never heard of that issue, was it basically not finding some of the packages that you installed with npm?

**Review by jackfales — APPROVED** [review]
Looks good!

**Pull request merged**